tests/rules/exec_properties/compare.bzl

```diff
@@ -0,0 +1,49 @@
+def _test_succeeds():
```

Comment:
Add license header and module docstring.

Reply:
done